owncloudApp/src/main/java/com/owncloud/android/ui/activity/UploadFilesActivity.java

```diff
@@ -409,19 +435,46 @@ public File getCurrentFolder() {
         return mCurrentDir;
     }
 
+    /**
+     * Sets the state of select all menu item based on the comparison
+     * between number of files that have been selected and the
+     * number of files present in current directory.
+     */
+    private void setSelectAllMenuItemState(){
+        if (mFileListFragment.getAdapter().getNoOfFilesInDir()
+                == mFileListFragment.getCheckedFilePaths().length) {
```

Comment:
Put the equal at the end of the previous line so that the reader knows that there's something below that
```
if (mFileListFragment.getAdapter().getNoOfFilesInDir() ==
mFileListFragment.getCheckedFilePaths().length) {
```